src/main/java/org/jenkinsCi/plugins/projectDescriptionSetter/DescriptionSetterWrapper.java

```diff
@@ -142,6 +148,11 @@ private String readFile(final FilePath projectDescriptionFile) {
             IOUtils.closeQuietly(in);
             IOUtils.closeQuietly(writer);
             throw new RuntimeException(ioe);
+        } catch (InterruptedException ie) {
+            IOUtils.closeQuietly(reader);
+            IOUtils.closeQuietly(in);
+            IOUtils.closeQuietly(writer);
+            throw new RuntimeException(ie);
```

Comment:
I doubt it is a good idea. The method is `private`, hence you could implement an `InterruptedException` propagation without wrapping into RunTime. I see the RuntieException approach is used above, but it is not what I would advertise